skflow/estimators/base.py

```diff
@@ -62,12 +62,15 @@ class TensorFlowEstimator(BaseEstimator):
         early_stopping_rounds: Activates early stopping if this is not None.
             Loss needs to decrease at least every every <early_stopping_rounds>
             round(s) to continue training. (default: None)
-
+        exponential_decay: Whether to apply exponential_decay to the learning_rate.
+                             A dict type containing the following keys: global_step(int),
+                             decay_steps(int), decay_rate(float), and staircase(bool).
```

Comment:
Good ideas. Consolidating them into learning_rate sounds better to me. This way there's no much change in trainer and it's much more flexible to use.